packages/components/CHANGELOG.md

```diff
@@ -300,6 +301,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Styles: `headingXXXLargeStyle` and `pds-heading-xxx-large`
 - Styles: `fontSizeHeadingXXLarge` and `$pds-font-size-heading-xx-large`
 
+#### 🤖 Property deprecations 🤖
```

Comment:
Wrong place now but also redundant with what we already have above. So it can be removed.